### ISSUE TITLE
Verify integration works with FarmXL

### ISSUE THREAD

**Issue by dalinicus** [dalinicus/homeassistant-aerogarden#15]
**Describe the bug**
This bug should be fixed with #2 as I did rework how multi gardens are identified, as well as switched to using config_id instead of mac_addr to identify an individual garden config.  

Someone from the thread https://github.com/jacobdonenfeld/homeassistant-aerogarden/issues/4 should be able to assist with this.

**Expected behavior**
All sensors are created and work for both halves of the garden.

**Comment by doctorkb** [comment]
I have the Farm 24 XL.

Aside from today with the API issues, it works correctly for me.

**Comment by dalinicus** [comment]
Amazing :) Thank you for your assistance.

Yeah, unfortunately the API is down enough that I had to put a badge for it on the repository.  Had a good run of stability these last few months, but apparently it couldn't last forever. 

I'll close this since the underlying code change has been validated on actual hardware.  Appreciate the help. 👍🏻

**Comment by doctorkb** [comment]
Oddly, the iOS app is still working. The last time I saw API issues, it was broken as well. Hopefully they aren't pulling a Chamberlain.

One thing that would be nice (but may not be feasible) would be to add an entity that indicated API issues -- rather than it just failing to load the integration altogether.  Just a thought.

**Comment by dalinicus** [comment]
Yeah, last time my aerogarden hardware couldn't connect either; the wifi symbol was yellow instead of green.   I'm currently between harvests, so I'm not sure if the hardware is having problems.... I do plan to start my next harvest sometime this week though.

If it stays down for two long, I'll whip out the packet sniffer and see if there's been any changes that would break the integration.

Attempting to show the status isn't a bad idea, but I'm not sure how that'll interact with how the sensors load.  I've created #31 to investigate.

**Comment by meeowgler** [comment]
Their API endpoint is returning response code 526 (Invalid SSL certificate): https://app3.aerogarden.com:8443/api/Admin/Login

When it's down, it appears that the app communicates with the garden locally? Need to take a closer look.

**Comment by dalinicus** [comment]
Yeah, I noticed that last night when I was poking around.  I opened a ticket with their support to let them know their certificate expired, but I haven't heard back yet.

Good to know.  Popped that URL into postman, but I'm getting an error trying to login via that endpoint.  May not be a drop in replacement for app3, or might not work at all.  I'll create an issue to investigate that URL further.

![image](https://github.com/dalinicus/homeassistant-aerogarden/assets/8620219/31fd0fcb-4ca9-4cd2-a757-7139fcd5c64a)

and yeah, their security is god awful.  I've cringed more than a few times when working on this project.
